scripts/overrides/index.js

```diff
@@ -30,66 +74,31 @@ function overrides(file, schema) {
     }
   }
 
-  // "/repos/{owner}/{repo}/compare/{base}...{head}" -> "/repos/{owner}/{repo}/compare/{basehead}"
-  if (schema.paths["/repos/{owner}/{repo}/compare/{basehead}"]) {
-    // update operation ID for new endpoint
-    schema.paths["/repos/{owner}/{repo}/compare/{basehead}"].get.operationId =
-      "repos/compare-commits-with-basehead";
-
-    // recove all endpoints
-    schema.paths["/repos/{owner}/{repo}/compare/{base}...{head}"] = {
-      get: /deref/.test(file)
-        ? require("./repos-compare-commits.deref.json")
-        : require("./repos-compare-commits.json"),
-    };
-  }
+  rewriteOperationId(schema, "/repos/{owner}/{repo}/compare/{basehead}", 'get', 'repos/compare-commits-with-basehead')
 
-  if (
-    schema.paths[
-      "/{owner}/{repo}/content_references/{content_reference_id}/attachments"
-    ]
-  ) {
-    schema.paths[
-      "/repos/{owner}/{repo}/content_references/{content_reference_id}/attachments"
-    ] =
-      schema.paths[
-        "/{owner}/{repo}/content_references/{content_reference_id}/attachments"
-      ];
-    delete schema.paths[
```

Comment:
This operation isn't in any current OpenAPI specs, so we should be safe to delete it.